### PR TITLE
Add color utils: hsl_to_rgb, hsv_to_rgb

### PR DESCRIPTION
**Pull request by stasm**

### CODE REVIEW

File: utils.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
-export function hex_to_vec(hex) {
+export function hex_to_rgb(hex) {
   if (hex.charAt(0) === '#') {
```
Comment (michalbe) [inline utils.js]: I had an idea in the back of my mind for quite a while right now and this PR looks like a nice place to finally implement it. What do you think about removing this block above, and skip the `#` sign from hex colors we're using? This is already happening in some places (like [this example](https://github.com/michalbe/cervus/blob/master/_example/animation/game.js#L7)).

File: utils.js [hunk 2]
```diff
@@ -25,3 +25,77 @@ export function vec_to_hex(vec) {
...
+ *
+ * @param   Number  h       The hue
+ * @param   Number  s       The saturation
+ * @param   Number  v       The value
+ * @return  Array           The RGB representation
+ */
+export function hsv_to_rgb(h, s, v) {
+  var r, g, b;
+
+  var i = Math.floor(h * 6);
```
Comment (michalbe) [inline utils.js]: `var i = ~~(h * 6);` should save some bytes.

File: utils.js (hunk 1 above)
Comment (stasm) [reply utils.js]: I don't have a strong feeling about this. The `#` is a sure sign that what you're seeing is a color and I would probably lean towards leaving it. `123456` doesn't look like much but `#123456` is quite different. In addition some editors highlight the hex colors with the color they define, which helps readability even more.

Comment (stasm) [reply utils.js]: Ha, even GitHub does this! #bada55.

Comment (michalbe) [reply utils.js]: Fair enough.

**Pull request merged**